schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+  id serial,
+  name VARCHAR(100),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg REAL,
```

Comment:
- [ ] I would suggest here you use `decimal` instead of `REAL` because `REAL` accepts only 1 number after the point and you are using more than that. Also, I think it is better when you have two numbers or more after the point